kasa/module.py

```diff
@@ -37,11 +37,20 @@ def data(self):
 
     def _add_feature(self, feature: Feature):
         """Add module feature."""
-        feat_name = f"{self._module}_{feature.name}"
+
+        def _slugified_name(name):
+            return name.lower().replace(" ", "_").replace("'", "_")
+
+        feat_name = _slugified_name(feature.name)
         if feat_name in self._module_features:
             raise KasaException("Duplicate name detected %s" % feat_name)
         self._module_features[feat_name] = feature
 
+    @property
+    def features(self) -> Dict[str, Feature]:
+        """Return the list of supported features."""
+        return self._module_features
```

Comment:
This is new API, follows the same idea as `Device`, perhaps it should be extracted into its own PR...

Reply:
What's the reason for this being being publicly exposed?  Aren't they available on the child device?

Reply:
Good question, I suppose there is no real need to expose this, not at least at the moment.